### ISSUE TITLE
Bug: Error if referenced class is detected as stereotype but is outside application

### ISSUE THREAD

**Issue by volkerkochnovatec** [NovaTecConsulting/stereotype-check#48]
Given this configuration with version 1.0.8:
```
	<stereotype id="repository">
		<postfix name="Repository" condition="sufficient" />
		<interface name="org.springframework.data.repository.Repository" condition="sufficient"/>
		<package name="^([a-z]+[a-z0-9\.]*)\.persistence(\.[a-z][a-z0-9]*)*$" condition="necessary" />
	</stereotype>
```

The import is marked as error:
import org.springframework.data.repository.Repository;

public interface JobRepository extends Repository<Job, Long> {

**Comment by volkerkochnovatec** [comment]
Maybe the problem was already solved with 2.0 because the implemented test does not show the error before changed code of stereotype checker

**Comment by ruben-burr** [comment]
As we do not know any more, what exactly the problem was, which shall be covered by this test, the ticket is closed without merging the pull request
